sympy/solvers/tests/test_inequalities.py

```diff
@@ -198,7 +198,7 @@ def test_reduce_abs_inequalities():
 
 def test_reduce_inequalities_general():
     assert reduce_inequalities(Ge(sqrt(2)*x, 1)) == And(sqrt(2)/2 <= x, x < oo)
-    assert reduce_inequalities(PurePoly(x + 1, x) > 0) == And(S(-1) < x, x < oo)
+    assert reduce_inequalities(PurePoly(x + 1, x) > 0) == And(-1 < x, x < oo)
```

Comment:
FYI: `-1 < x` will become `x > -1` whereas `S.NegativeOne < x` will remain unchanged. But apparently (since tests pass) `And` makes the args canonical.

Reply:
Oh, I thought tests were passing but they aren't. So I imagine that this is what caused the failure.

Reply:
Yes, this was causing a bit of the failures.